src/pages/[locale]/start/index.tsx

```diff
@@ -0,0 +1,135 @@
+import { GetStaticProps } from "next"
+import dynamic from "next/dynamic"
+import { useRouter } from "next/router"
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query"
+
+import { BasePageProps, Lang, Wallet } from "@/lib/types"
+
+import { Image } from "@/components/Image"
+import MainArticle from "@/components/MainArticle"
+import PageMetadata from "@/components/PageMetadata"
+import StartWithEthereumFlow from "@/components/StartWithEthereumFlow"
+import ShareModal from "@/components/StartWithEthereumFlow/ShareModal"
+
+import { existsNamespace } from "@/lib/utils/existsNamespace"
+import { getLastDeployDate } from "@/lib/utils/getLastDeployDate"
+import { getLocaleTimestamp } from "@/lib/utils/time"
+import { getRequiredNamespacesForPage } from "@/lib/utils/translations"
+import { getNewToCryptoWallets } from "@/lib/utils/wallets"
+
+import { DEFAULT_LOCALE, LOCALES_CODES } from "@/lib/constants"
+
+import loadNamespaces from "@/i18n/loadNamespaces"
+import HeroImage from "@/public/images/heroes/developers-hub-hero.jpg"
+import ManDogeImage from "@/public/images/start-with-ethereum/man-doge-playing.png"
+
+// Dynamically import Wagmi/RainbowKit components
+const WalletProviders = dynamic(() => import("@/components/WalletProviders"), {
+  ssr: false,
+})
+
+const queryClient = new QueryClient()
+
+export async function getStaticPaths() {
+  return {
+    paths: LOCALES_CODES.map((locale) => ({ params: { locale } })),
+    fallback: false,
+  }
+}
+
+export const getStaticProps = (async ({ params }) => {
+  const { locale = DEFAULT_LOCALE } = params || {}
+
+  const lastDeployDate = getLastDeployDate()
+  const lastDeployLocaleTimestamp = getLocaleTimestamp(
+    locale as Lang,
+    lastDeployDate
+  )
+
+  const requiredNamespaces = getRequiredNamespacesForPage("/start")
+
+  const contentNotTranslated = !existsNamespace(
+    locale! as string,
+    requiredNamespaces[2]
+  )
+
+  const messages = await loadNamespaces(locale as string, requiredNamespaces)
+
+  const newToCryptoWallets = getNewToCryptoWallets()
+
+  return {
+    props: {
+      messages,
+      contentNotTranslated,
+      lastDeployLocaleTimestamp,
+      newToCryptoWallets,
+    },
+  }
+}) satisfies GetStaticProps<BasePageProps>
+
+const StartWithCryptoPage = ({
+  newToCryptoWallets,
+}: {
+  newToCryptoWallets: Wallet[]
+}) => {
+  const router = useRouter()
+
+  return (
+    <QueryClientProvider client={queryClient}>
+      <WalletProviders locale={router.query.locale as string}>
```

Comment:
We should use now the `useLocale` hook to get it. Pushing a commit for this as well.